### PR TITLE
Beta3 analytics

### PR DESCRIPTION
**Pull request by whitetiger1399**
## Proposed changes
1. Font family changed to `Inter`
2. Overview tab icons mapping on dashboard table.
3. Chaos event specific sub data filtering.
4. Color changed for Tooltip for graph buttons.
5. Accordion icon changed.
6. Accordion from overview tab file shifted to the components folder
7. Zooming bug fix for chaos metrics event data. 
8. QuickActionCardProps and EventMetric interface imported from Litmus-ui
9. One translation added for the analytics page

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/components/ToolTip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Theme, withStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+
+const ToolTip = withStyles((theme: Theme) => ({
+  tooltip: {
+    backgroundColor: theme.palette.background.paper,
+    color: theme.palette.text.primary,
+    boxShadow:
+      '0rem 0.128rem 0.384rem rgba(0,0,0,0.1), 0rem 0.682rem 1.54rem rgba(0,0,0,0.13)',
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/ToolTip/index.tsx:9]: can we take colours from litmus-ui theme file?

File: litmus-portal/frontend/src/components/ToolTip/index.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    fontSize: 10,
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/components/ToolTip/index.tsx:10]: unit ?

File: litmus-portal/frontend/src/utils/promUtils.ts [hunk 3]
```diff
@@ -524,67 +529,49 @@ export const chaosEventDataParserForPrometheus = (
...
+                      : '--',
+                  date: elem ? elem.lastUpdatedTimeStamp : 0,
+                },
+                {
+                  subDataName: 'Probe Success Percentage',
+                  value: elem ? elem.probeSuccessPercentage : '--',
+                  date: elem ? elem.lastUpdatedTimeStamp : 0,
+                },
+                {
+                  subDataName: 'Experiment Verdict',
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/utils/promUtils.ts:561]: add translation

File: litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/styles.ts [hunk 4]
```diff
@@ -115,8 +117,9 @@ const useStyles = makeStyles((theme: Theme) => ({
     '& p': {
       paddingRight: theme.spacing(2),
     },
-    marginRight: theme.spacing(5),
-    paddingRight: theme.spacing(5),
+    padding: 0,
+    marginRight: theme.spacing(5.75),
+    paddingRight: theme.spacing(5.75),
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/styles.ts:122]: use one padding style specify all in one line

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/components/ToolTip/index.tsx (hunk 1 above)
Comment (whitetiger1399) [reply litmus-portal/frontend/src/components/ToolTip/index.tsx:9]: Tooltip specific colors `rgba` are not defined in Litmus-ui

File: litmus-portal/frontend/src/utils/promUtils.ts (hunk 3 above)
Comment (whitetiger1399) [reply litmus-portal/frontend/src/utils/promUtils.ts:561]: This is neither a react function nor a custom hook how to add a translation?

Comment (whitetiger1399) [reply litmus-portal/frontend/src/utils/promUtils.ts:561]: The only way to add translation here it by adding translation path here and then while constructing the chaos_data we have to map again over every element and then add a translation hook for all sub-data

File: litmus-portal/frontend/src/components/ToolTip/index.tsx (hunk 1 above)
Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/components/ToolTip/index.tsx:9]: let's try to take it from theme.shadows[<elevation>]
elevation would be 0...24 any number you seem appropriate

File: litmus-portal/frontend/src/components/ToolTip/index.tsx (hunk 2 above)
Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/components/ToolTip/index.tsx:10]: unit is not needed...default is px
but here font size in rem would be better

File: litmus-portal/frontend/src/config/index.ts [hunk 5]
```diff
@@ -13,7 +13,7 @@ if (
 ) {
   authURL = `${window.location.protocol}//${window.location.hostname}:3000`;
   apiURL = `${window.location.protocol}//${window.location.hostname}:8080`;
-  sockURL += `//${window.location.hostname}:8080`;
+  sockURL += `//${window.location.hostname}:8080`;
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/config/index.ts:16]: extra space added

File: litmus-portal/frontend/src/utils/promUtils.ts (hunk 3 above)
Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/utils/promUtils.ts:561]: subData is used in litmus-ui...there we don't have a translation file to process them @ishangupta-ds

File: litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/styles.ts (hunk 4 above)
Comment (arkajyotiMukherjee) [reply litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/styles.ts:122]: If the requirement is setting all padding to 0 and changing only one...this is fine. @ishangupta-ds 
but is the `padding: 0` line required? @whitetiger1399

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/utils/promUtils.ts (hunk 3 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/utils/promUtils.ts]: Have suggested a way to @whitetiger1399 cc: @arkajyotiMukherjee

**Review by ishangupta-ds — APPROVED** [review]

**Comment by ishangupta-ds** [comment]
This PR would break the existing integration, conditionally approved, backend fixes or thresholding is needed. @whitetiger1399

File: litmus-portal/frontend/src/config/index.ts [hunk 6]
```diff
@@ -11,9 +11,12 @@ if (
...
   process.env.NODE_ENV.trim() === 'test'
 ) {
-  authURL = `${window.location.protocol}//${window.location.hostname}:3000`;
-  apiURL = `${window.location.protocol}//${window.location.hostname}:8080`;
-  sockURL += `//${window.location.hostname}:8080`;
+  // authURL = `${window.location.protocol}//${window.location.hostname}:3000`;
+  // apiURL = `${window.location.protocol}//${window.location.hostname}:8080`;
+  // sockURL += `//${window.location.hostname}:8080`;
+  authURL = `http://a95867befb41a44e8b48c4e62e4f1f46-345793451.us-east-2.elb.amazonaws.com:9003`;
+  apiURL = `http://a95867befb41a44e8b48c4e62e4f1f46-345793451.us-east-2.elb.amazonaws.com:9002`;
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/config/index.ts:18]: can you change these back

**Review by arkajyotiMukherjee — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/config/index.ts [hunk 7]
```diff
@@ -11,9 +11,12 @@ if (
   process.env.NODE_ENV.trim() === 'development' ||
   process.env.NODE_ENV.trim() === 'test'
 ) {
-  authURL = `${window.location.protocol}//${window.location.hostname}:3000`;
-  apiURL = `${window.location.protocol}//${window.location.hostname}:8080`;
-  sockURL += `//${window.location.hostname}:8080`;
+  // authURL = `${window.location.protocol}//${window.location.hostname}:3000`;
+  // apiURL = `${window.location.protocol}//${window.location.hostname}:8080`;
+  // sockURL += `//${window.location.hostname}:8080`;
+  authURL = `http://a95867befb41a44e8b48c4e62e4f1f46-345793451.us-east-2.elb.amazonaws.com:9003`;
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/config/index.ts]: Can we revert it? @whitetiger1399

File: litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/Tables/dataSource.tsx [hunk 8]
```diff
@@ -67,68 +68,70 @@ const TableDataSource: React.FC<TableDataSourceProps> = ({
...
-                      {t('analyticsDashboard.seeDetails')}
+            <TableBody>
+              {dataSourceList.slice(0, 3).map((singleDataSource) => (
+                <TableRow
+                  key={singleDataSource.ds_id}
+                  className={classes.tableRow}
+                >
+                  <TableCell scope="row" className={classes.tableRowHeader}>
+                    <div className={classes.dataSourceTableHeader}>
+                      {singleDataSource.health_status === 'Active' ? (
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/Tables/dataSource.tsx:79]: Can we have this status in the constants file? we are doing the same for workflow status

File: litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/Tables/dataSource.tsx [hunk 9]
```diff
@@ -67,68 +68,70 @@ const TableDataSource: React.FC<TableDataSourceProps> = ({
...
+                        <LightPills
+                          variant="success"
+                          label={singleDataSource.health_status}
+                        />
+                      ) : singleDataSource.health_status === 'Not Ready' ? (
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/Tables/dataSource.tsx:84]: Same as above

**Review by Jonsy13 — CHANGES_REQUESTED** [review]
Thanks, @whitetiger1399, PTAL for minor comments

File: litmus-portal/frontend/src/config/index.ts (hunk 6 above)
Comment (whitetiger1399) [reply litmus-portal/frontend/src/config/index.ts]: yes it was a temporary change

File: litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/Tables/dataSource.tsx (hunk 9 above)
Comment (whitetiger1399) [reply litmus-portal/frontend/src/views/AnalyticsDashboard/Overview/Tables/dataSource.tsx]: Not sure of having a separate file but I have added the constants in the same file itself

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
Please add the previous logic of latest run data @whitetiger1399 for the promutils file.

**Comment by whitetiger1399** [comment]
> Please add the previous logic of latest run data @whitetiger1399 for the promutils file.
The date key in the subData is required. Simply resetting the promutils file will not work

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]
please check package lock changes

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**